llpc/context/llpcGraphicsContext.cpp

```diff
@@ -222,11 +222,11 @@ void GraphicsContext::setTcsInputVertices(Module *tcsModule) {
 Options GraphicsContext::computePipelineOptions() const {
   Options options = PipelineContext::computePipelineOptions();
 
-  options.enableUberFetchShader =
-      reinterpret_cast<const GraphicsPipelineBuildInfo *>(getPipelineBuildInfo())->enableUberFetchShader;
+  auto pipelineInfo = reinterpret_cast<const GraphicsPipelineBuildInfo *>(getPipelineBuildInfo());
```

Comment:
static_cast instead of reinterpret_cast (always use the "weakest" cast possible)